### DISCUSSION TITLE
Problemi di installazione di HA Core su Raspberry Pi4

### DISCUSSION THREAD

**Discussion by Rupe64** [indomus/forum#2280] [HUB: Home Assistant]
Salve a tutti,
sto cercando di installare HA Core come da titolo seguendo la vostra guida
https://indomus.it/guide/come-installare-e-configurare-home-assistant-su-un-raspberry-pi-gia-in-uso/
ma incappo in un messaggio di errore all'installazione eseguendo il comando
```
hass -v
```
Ho il forte sospetto che il problema derivi da una situazione del tutto simile a quella della discussione
https://github.com/indomus/forum/discussions/1129
relativa ad un problema nell'installazione delle librerie Python3, ed in particolare delle ```libjpeg, libturbojpeg0```
Di seguito riporto la parte finale dell'output del comando dell'installazione delle librerie Python
```
...
The following additional packages will be installed:
  automake autotools-dev libjpeg62-turbo-dev libsigsegv2 m4
Suggested packages:
  autoconf-archive gnu-standards autoconf-doc libtool gettext m4-doc
The following NEW packages will be installed:
  autoconf automake autotools-dev libffi-dev libjpeg-dev libjpeg62-turbo-dev
  libsigsegv2 libturbojpeg0 m4
0 upgraded, 9 newly installed, 0 to remove and 0 not upgraded.
Need to get 359 kB/1,924 kB of archives.
After this operation, 5,451 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Err:1 http://raspbian.raspberrypi.org/raspbian buster/main armhf libjpeg62-turbo-dev armhf 1:1.5.2-2+b1
  404  Not Found [IP: 93.93.128.193 80]
Err:2 http://raspbian.raspberrypi.org/raspbian buster/main armhf libjpeg-dev all 1:1.5.2-2
  404  Not Found [IP: 93.93.128.193 80]
Err:3 http://raspbian.raspberrypi.org/raspbian buster/main armhf libturbojpeg0 armhf 1:1.5.2-2+b1
  404  Not Found [IP: 93.93.128.193 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/libj/libjpeg-turbo/libjpeg62-turbo-dev_1.5.2-2+b1_armhf.deb  404  Not Found [IP: 93.93.128.193 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/libj/libjpeg-turbo/libjpeg-dev_1.5.2-2_all.deb  404  Not Found [IP: 93.93.128.193 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/libj/libjpeg-turbo/libturbojpeg0_1.5.2-2+b1_armhf.deb  404  Not Found [IP: 93.93.128.193 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```
Il problema che non risolvo è che non riesco a trovare il modo di installarle...
Qualcuno ha qualche suggerimento?
Grazie!

**Comment by snakuzzo** [comment]
Se posso darti un consiglio...lascia perdere questo tipo di installazione.
Con i vari aggiornamenti di core le versioni di Python supportate cambiano.
Inoltre, le versioni "latest" (concedimi il termine) di Python disponibili di default tramite i package manager degli OS variano anche a seconda della versione dello stesso OS.
Poi mettici che quando parte HA, a seconda dei moduli python necessari, necessità di compilare alcuni pacchetti, che a loro volta necessitano di pacchetti a livello di OS.

Per evitare ogni volta questa rottura, ti consiglio di valutare l'installazione di core con docker e docker-compose

**Reply by Rupe64** [reply]
Grazie per la risposta.
Temevo che il problema potesse essere di questo tipo... ma ho provato. Seguirò la strada del docker.
Grazie ancora.

**Reply by snakuzzo** [reply]
Sì, ti conviene.
Ovvio che c'è sempre la soluzione per l'installazione che stavi facendo, ma se hai pazienza, ti consiglio docker
